libs/account-lib/src/blockchain/blockchain.service.ts

```diff
@@ -648,4 +648,15 @@ export class BlockchainService implements OnApplicationBootstrap, OnApplicationS
   public decodeTransaction(encodedExtrinsic: string) {
     return this.api.tx(encodedExtrinsic);
   }
+
+  public getNetworkType(): 'mainnet' | 'testnet-paseo' | 'unknown' {
```

Comment:
I went with a typed string instead of enum, but open to changing.

Reply:
I'm fine either way, but would prefer to see it exported as a reusable type:
```
export type NetworkType = 'mainnet' | 'testnet-paseo' | 'unknown';
```